src/pages/Landing.tsx

```diff
@@ -70,6 +70,27 @@ const SubText = styled.h3`
   }
 `
 
+const CTAButton = styled(BaseButton)`
+  width: 288px;
+  padding: 16px;
+  border-radius: 24px;
+  color: ${({ theme }) => theme.white};
+
+  &:hover {
+    opacity: 50%;
+  }
+`
+
+const ButtonCTA = styled(CTAButton)`
+  background: linear-gradient(10deg, rgba(255, 0, 199, 1) 0%, rgba(255, 159, 251, 1) 100%);
+  border: none;
+`
+
+const ButtonCTASecondary = styled(CTAButton)`
+  background: none;
```

Comment:
I am pretty sure I've seen some styles like SecondaryButton or Outline in the code that have no background, so you could potentially use that one instead?

Reply:
*ButtonOutlined

Reply:
probably true but i feel like that'll be more work that isn't worth it right now, until we have solidified buttons